cmd/main.go

```diff
@@ -203,10 +203,11 @@ func main() {
 
 	var openshiftVersionsMap models.OpenshiftVersions
 	if Options.OpenshiftVersions == "" {
-		log.Fatal("OpenShift versions is empty")
+		log.Info("OpenShift versions is empty - using OsImages and ReleaseImages")
```

Comment:
```suggestion
		log.Info("OpenShift versions is empty - using default values for OS and release images")
```

Reply:
It's not necessarily the default values... as os_images/release_images can still be specified. But I'll rephrase to make it clearer.